demos/text_detection_demo/src/text_detection.cpp

```diff
@@ -100,6 +100,34 @@ std::vector<cv::RotatedRect> maskToBoxes(const cv::Mat &mask, float min_area, fl
     return bboxes;
   }
 
+std::vector<cv::RotatedRect> coordToBoxes(const std::vector<float> &coords, float min_area, float min_height,
+                                         cv::Size image_size) {
+    std::vector<cv::RotatedRect> bboxes;
+    int max_bbox_idx = coords.size() / 5;
+    float x_scale = image_size.width / 704.0;
+    float y_scale = image_size.height / 704.0;
+    //TODO: Find a way not to hardcode the certain input_shape (704, 704)
+    for (int i = 1; i < max_bbox_idx; i++) {
```

Comment:
will you consider all boxes in such range?

Reply:
This max_bbox_idx value is equal to the max number of prediction per image (in the horizontal-text-detection model it is 100). Some of them are insignificant, so i filter it out [here](https://github.com/GalyaZalesskaya/open_model_zoo/blob/da991e8cb0b82c714adad571291725c779dead0a/demos/text_detection_demo/src/text_detection.cpp#L115).
If I misunderstood the question, please say